scss/components/_case-card.scss

```diff
@@ -60,6 +60,11 @@
     overflow: hidden;
     white-space: nowrap;
   }
+
+  .civicase__contact-name,
+  .civicase__contact-name-additional {
+    max-width: 100px;
```

Comment:
Should we document this in a variable?

Reply:
I feel this is not necessary to document in a variable, because
1. This is not used more than once,
2. This is self explanatory, that we want to set the max width of the Contact name when used inside Case Card.